### ISSUE TITLE
Collapse synonym table into taxonid

### ISSUE THREAD

**Issue by cboettig** [cboettig/taxald#12]
For instance: 

```
all_itis <- 
  taxa_tbl("itis", "synonyms") %>% 
  select(name, id, rank, update_date) %>% 
  mutate(type = "synonym") %>% 
  union( select(taxa_tbl("itis", "taxonid"))%>% 
           mutate(type = "accepted") ) 
```

Then to get the accepted names, we need another function or another option on `ids()`.  

If `df` is the return from joining `all_itis` with my species, then:

```
all_itis %>% 
  filter(type == "accepted") %>% 
  select(id, accepted_name = name) %>% 
  right_join(df)
```

**Comment by karinorman** [comment]
@cboettig It looks like there are not column names for the itis flat files, is it a download problem?

**Comment by cboettig** [comment]
hmm... could be an export problem, or could be just the way the database is made, lemme take a look

**Comment by cboettig** [comment]
Implemented in #14, closing
